examples/05-Retrieval-Model.ipynb

```diff
@@ -999,7 +999,9 @@
    "source": [
```

Comment:
<p>thanks for the PR.  we need to rephrase this sentence <code>The method extracts unique rows from a specified dataset (train_raw) based on a specified id-column tags (ITEM and ITEM_ID)</code>  since in the example, we do not feed raw train set into this utility function, we are feeding transformed <code>train</code> dataset object.</p><p> </p><p>so better to write as follows instead:</p><p> </p><p><code>unique_rows_by_features</code> : A utility function allows extracting both unique user and item features tables as Merlin <code>Dataset</code> object that can easily be converted to a cuDF data frame. The function extracts unique rows from a specified dataset (transformed train set) based on a specified id-column tags (<code>ITEM</code> and <code>ITEM_ID</code>).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/967/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>I meant  write here  the following:</p><p> </p><p> </p><p> </p><p><code>unique_rows_by_features</code>&nbsp;: A utility function allows extracting both unique user and item features tables as Merlin&nbsp;<code>Dataset</code>&nbsp;object that can easily be converted to a cuDF data frame. The function extracts unique rows from a specified dataset (transformed train set) based on a specified id-column tags (<code>ITEM</code>&nbsp;and&nbsp;<code>ITEM_ID</code>).</p>